elements/lisk-api-client/src/types.ts

```diff
@@ -33,18 +33,13 @@ export interface JSONRPCResponse<T> {
 
 export type JSONRPCMessage<T> = JSONRPCNotification<T> | JSONRPCResponse<T>;
 
-export interface EventInfoObject<T> {
-	readonly module: string;
-	readonly name: string;
-	readonly data: T;
-}
-export type EventCallback<T> = (event: EventInfoObject<T>) => void | Promise<void>;
+export type EventCallback = (event?: Record<string, unknown>) => void | Promise<void>;
```

Comment:
Can we make generic as it was earlier, so we can pass on the type of data the event handler is gonna receive?